packages/db/index.js

```diff
@@ -121,7 +121,9 @@ export class DBClient {
     return {
       id: userData.id,
       inserted: userData.inserted,
-      issuer: userData.issuer
+      issuer: userData.issuer,
+      name: userData.name,
+      email: userData.email
```

Comment:
Do these fields actually come back from the sql query? I don't think they will based on the underlying `upsert_user` [https://github.com/web3-storage/web3.storage/blob/main/packages/db/postgres/functions.sql#L449]

(I think if you wrote a test assertion that they are defined after calling `upsertUser`, it might fail).

Regardless, though, can we do this in a way that doesn't involve always returning this from upsertUser? It's an upsert, the caller of the function just passed in those fields, so it seems like there's no point in making the return type include them (and the underlying query more advanced).

Reply:
It was coincidental that this worked at all, since the name and email were coming from auth instead of the returned value here.  I've since updated this PR to align with your notion doc.  Thanks!